framework/doc/content/source/transfers/MultiAppInterpolationTransfer.md

```diff
@@ -1,8 +1,25 @@
 # MultiAppInterpolationTransfer
 
-The MultiAppInterpolationTransfer transfers the source variable to the nearest node on the
+The `MultiAppInterpolationTransfer` transfers the source variable to the nearest node on the
 target mesh using mesh interpolation, including the ability to utilize the displaced
-configuration for either or both the source and target.
+configuration for either or both the source and target.  
+
+Nodal transfers using the default settings for this interpolation scheme, `interp_type=inverse_distance` and `num_points=3`, are 
+best suited for the interpolation of point cloud data in the source mesh onto a target mesh.  
+The [MultiAppMeshFunctionTransfer](/MultiAppMeshFunctionTransfer.md) may be a better choice for nodal transfers 
+between two meshes because the element shape functions will be used in the transfer.
+Using `MultiAppInterpolationTranser` with the default interpolation settings for mesh to mesh nodal transfers of data 
+can lead to a parallel inconsistency that arises when the point being interpolated to on the
+target mesh is exactly the same distance away from more than `num_points` on the source mesh.  
+
+This node-to-node transfer inconsistency could occur on a structured mesh if the target mesh is a refined version of the source mesh.  
+In this scenario, a target node will be placed equidistant from several source nodes.  
+For a 2D structured mesh of quadrilaterals, the refined mesh's target node would be equidistant from 4 nodes on the source mesh.  
+With the default `num_points=3`, the three points chosen from the four equidistant nodes on the source mesh will be arbitrary and has been shown to be dependent on the parallel decomposition of the mesh.  
+This inconsistency goes away for this case by increasing `num_points=4` so that all of the nodes in the element are used for the interpolation.  
+
+For unstructured meshes, this parallel inconsistency *should not* be an issue. 
```

Comment:
Parallel inconsistency will happen if a target point has the same different distance to different groups of source points, regardless of meshes

Reply:
I changed all of the "Parallel inconsistency" to "not deterministic"